sbnobj/Common/Reco/MVAPID.h

```diff
@@ -0,0 +1,24 @@
+// Class for holding the result of an MVA PID
+
+#ifndef sbncode_MVAPID_H
+#define sbncode_MVAPID_H
+
+#include <map>
+#include <string>
+
+namespace sbn {
+class MVAPID {
+  public:
+  MVAPID() {}
+
+  std::map<int, float> mMVAScoreMap; // MVA Scores: Key is PDG of hypothesis and value is the MVA score
+
+  void AddScore(int pdg, float score); // Function to call to add a new score to the map
+
+  std::pair<int, float> BestIter() const;                             // Funtion to return the map iterator with the highest score
```

Comment:
Typo in comment. Also, we know they're functions. How about just "Return the map entry with the highest score"?